core/examples/luigi-sample-angular/src/luigi-config/extended/navigation.js

```diff
@@ -234,8 +234,6 @@ class Navigation {
     defaultLabel: 'Select Environment ...',
     parentNodePath: '/environments', // absolute path
     lazyloadOptions: true, // load options on click instead on page load
-    preserveSubPathOnSwitch: true,
```

Comment:
maybe these changes are not relevant to this PR. the same for `core/src/navigation/services/navigation.js`